### ISSUE TITLE
Run model with best hyperparams

### ISSUE THREAD

**Issue by hbandukw** [hbandukw/ClimateDebunk#9]
Trial 21 finished with value: 0.8999179655455292 and parameters: {'learning_rate': 0.0001932713713098696, 'num_trainable_layers': 1, 'dropout_rate': 0.4517023694187439, 'batch_size': 32, 'step_size': 8, 'gamma': 0.7710877288358753, 'epochs': 5}. Best is trial 21 with value: 0.8999179655455292.

results: Final Model Accuracy: 0.6218
